src/ActionSheet/ActionGroup.tsx

```diff
@@ -97,11 +129,13 @@ export default class ActionGroup extends React.Component<Props> {
 
       optionViews.push(
         <TouchableNativeFeedbackSafe
+          ref={i === 0 ? focusViewOnRender : undefined}
```

Comment:
I'm not sure if it'd be desirable to make this behavior configurable via a prop (or disabled by default, for example)

Reply:
Added an autoFocus prop